src/components/drawing/index.js

```diff
@@ -482,6 +670,16 @@ drawing.singlePointStyle = function(d, sel, trace, fns, gd) {
             if(!gradientInfo[gradientType]) gradientType = 0;
         }
 
+        var getPatternAttr = function(mp, i, dflt) {
```

Comment:
Non-blocking: 
It looks like you could also move this function above `drawing.pointStyle`.

Reply:
Fixed in 51614783013bdc48ea9d8076f44a31a8da95b5a5